elixir/mix.exs

```diff
@@ -5,8 +5,7 @@ defmodule Firezone.MixProject do
     [
       name: :firezone,
       apps_path: "apps",
-      # mark:automatic-version
-      version: "1.20231001.0",
+      version: String.trim(File.read!("VERSION")),
```

Comment:
```suggestion
      version: String.trim(File.read!("../VERSION")),
```

Reply:
@jamilbk `VERSION` and `mix.exs` are in the same folder (`elixir/`).

Reply:
See this comment: https://github.com/firezone/firezone/pull/2257#discussion_r1347806432